public/static/app/perfdiscovery/PerfDiscoveryDataService.js

```diff
@@ -139,56 +154,58 @@ mciModule.factory('PerfDiscoveryDataService', function (
   //   },
   //   {...}
   // ]
-  function onProcessData(data) {
-    var now = {},
-      baseline = {},
-      history = {}
-
-    _.each(data, function (d) {
-      // Skip empty items
-      if (d == null) return
-
-      // TODO add group validation instead of individual
-      // Process current (revision) data
-      if (d.current && d.current.data) {
-        // Copy storageEngine to the context
-        d.ctx.storageEngine = d.current.data.storageEngine || '(none)'
-
-        _.each(d.current.data.results, function (result) {
-          processItem(result, now, d.ctx)
-        })
-      }
-
-      // Process baseline data
-      if (d.baseline && d.baseline.data) {
-        _.each(d.baseline.data.results, function (result) {
-          processItem(result, baseline, d.ctx)
-        })
-      }
+  function onProcessData(metricName, data) {
```

Comment:
the data parameter is no longer accessed. Should it be removed?

Reply:
Yep, done